master/getting-started/openshift/installation.md

```diff
@@ -1,10 +1,28 @@
 ---
 title: Installing Calico on OpenShift
+redirect_from: latest/getting-started/openshift/installation
 ---
 
-Installation of Calico in OpenShift is integrated in openshift-ansible v3.6.
+Calico has been integrated with both OpenShift Origin and Red Hat OpenShift Container Platform, and is deployed using the standard OpenShift-ansible installation process (which includes roles to deploy and configure Calico).
+
+Calico replaces openshift-sdn and the network policy implementation for openshift-sdn, but functions with other standard OpenShift artifacts including kube-proxy, OpenShift router (HAProxy), Registry, and DNS. 
+
+Deployment of Calico with OpenShift is done using the standard openshift-ansible deployment process, but with the appropriate Calico variables set in the host inventory file for openshift-ansible as described below. Please ensure that all OpenShift prerequisites are met and host preparation steps are performed prior to starting the OpenShift install, as described in [OpenShift Container Platform documentation](https://access.redhat.com/documentation/en-us/openshift_container_platform/3.6/html-single/installation_and_configuration/#install-config-install-prerequisites) or [OpenShift Origin documentation](https://docs.openshift.org/latest/install_config/install/prerequisites.html).
+
+Calico deployment roles have been integrated with openshift-ansible since v3.6.
 The information below explains the variables which must be set during
-during the standard [Advanced Installation](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables).
+during the standard [OpenShift Origin Advanced Installation](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables) or [OpenShift Container Platform Advanced Installation](https://access.redhat.com/documentation/en-us/openshift_container_platform/3.6/html-single/installation_and_configuration/#install-config-install-advanced-install).
+
+## Etcd Backend for Calico
+There are two alternative options for the Datastore backend for Calico within OpenShift deployments:
+
+- Shared Etcd with OpenShift
+This is suitable for non-production deployments of OpenShift, and involves Calico using the same Etcd server instances as are used by OpenShift (Calico uses a different part of the Etcd namespace than OpenShift). 
+
+- Dedicated Etcd for Calico (Bring-your-own Etcd)
```

Comment:
"slack" --> "Slack"

Also, I would make Slack a hyperlink so they can click it and get to our Slack.